pkg/entry/mongo/document.go

```diff
@@ -122,10 +191,10 @@ func (d *Document) Populate(ctx context.Context, record map[interface{}]interfac
 		return fmt.Errorf("parse %s: %w", PlatformIDKey, err)
 	}
 
-	return d.generateObjectID()
```

Comment:
Why do not generate object ID here?
What is the goal of duplicating this line in `JobLogDocument.Populate()` and `AppLogDocument.Populate()`?

I suggest to keep `return d.generateObjectID()` here.

Reply:
The generateObjectID() is generated from the serialization of the document. It needs to be used on a fully populate document